src/odemis/dataio/tiff.py

```diff
@@ -2722,3 +2752,210 @@ def _convertTFStoOdemisMD(metadata: str) -> dict:
         logging.warning(f"Failed to parse ThermoFisher metadata: {model.MD_DET_TYPE} - {e}")
 
     return md
+
+def _parseZEISSMetadata(metadata) -> dict:
```

Comment:
Let's drop the "ZEISS" in all capital (in function names), it's really annoying to read.